admin_manual/configuration_database/db_conversion.rst

```diff
@@ -2,9 +2,9 @@
 Converting Database Type
 ========================
 
-You can convert a SQLite database to a more performing MySQL, MariaDB or 
-PostgreSQL database with the ownCloud command line tool. SQLite is good for 
-testing and simple single-user ownCloud servers, but it does not scale for multiple-user sites.
+SQLite is good for testing ownCloud, as well as small, single-user, ownCloud servers. 
+But, **it does not scale** for large, multi-user sites.
+If you have an existing ownCloud installation which uses SQLite, and you want to convert to a more performing database, such as *MySQL*, *MariaDB* or *PostgreSQL*, you can use :ref:`the ownCloud command line tool: occ <database_conversion_label>`. 
```

Comment:
Thanks for the extra vote on that. I wasn't sure whether to make that exact change, or to leave it.